addons/website_sale/models/res_config_settings.py

```diff
@@ -37,6 +37,7 @@ def _default_recovery_mail_template(self):
 
     cart_recovery_mail_template = fields.Many2one('mail.template', string='Cart Recovery Email', domain="[('model', '=', 'sale.order')]",
                                                   default=_default_recovery_mail_template, related='website_id.cart_recovery_mail_template_id')
+    confirmation_template_id = fields.Many2one('mail.template', string='Confirmation Email', domain="[('model', '=', 'sale.order')]", config_parameter='website_sale.ecommerce_confirmation_template')
```

Comment:
Why having 2 configuration for basically the same thing ? At least update string accordingly and provide some helpers to explain why you need separate templates.